skyportal/tests/frontend/test_observing_planner.py

```diff
@@ -50,7 +50,7 @@ def test_source_is_added_to_observing_run_via_frontend(
 
     driver.scroll_to_element_and_click(submit_button)
     driver.get(f"/run/{red_transients_run.id}")
-    driver.wait_for_xpath(f'//*[text()="{public_source.id}"]')
+    driver.wait_for_xpath(f'//*[text()="{public_source.id}"]', timeout=20)
```

Comment:
this and the similar invocations later in this file are a poor man's solution (for now) to the root cause of the flakiness of these tests, which is described in #1102

Reply:
basically, the page takes a long time to load right now because there are some non trivial flops that have to be done on the backend handler

Reply:
eventually these will be cached and we can remove these longer timeouts

Reply:
Great, let's add that note of your here so we know about it.

Reply:
done